lib/datasource.js

```diff
@@ -513,6 +513,9 @@ DataSource.prototype.setupDataAccess = function(modelClass, settings) {
       idProp.type = idType;
       modelClass.definition.rawProperties[idName].type = idType;
       modelClass.definition.properties[idName].type = idType;
+      if (['oracle', 'mysql'].indexOf(this.connector.name) !== -1) {
```

Comment:
@Amir-61 Thank you lol, it's not finished, I haven't checked all auto-increment datasources. I created this pr to see how many repos fail due to my change to estimate the scale i need to take care of.